packages/tc-schema-validator/validation/index.js

```diff
@@ -108,5 +108,5 @@ const fail = () => {
 
 process.on('unhandledRejection', error => {
 	console.error(error);
-	fail();
+	// fail();
```

Comment:
Temporary code whilst @nl-ria applies the validator upgrade - forces all yamls to be treated as valid.